lib/u3d/commands.rb

```diff
@@ -103,7 +103,8 @@ def list_available(options: {})
       def install(args: [], options: {})
         version = specified_or_current_project_version(args[0])
 
-        os = U3dCore::Helper.operating_system
+        UI.user_error!("install can only work with current operating system. You specified #{options[:operating_system]}") if options[:install] && options[:operating_system]
```

Comment:
I would maybe specify the use of the --no-install here

Reply:
At this point I don't know if the user specified install explicitly or if it was the default. 

Is this better?
```
UI.user_error!("You cannot use the --operating_system and the --install options together") if options[:install] && options[:operating_system]
```

Reply:
Much better indeed!